firmwares/aeotec/ZWA024-A.json

```diff
@@ -0,0 +1,31 @@
+{
+	"devices": [
+		{
+			"brand": "aeotec",

```

Comment:
Is this supposed to be lowercase? We call it `Aeotec` (uppercase) everywhere else.

Reply:
I kinda followed the flow of other examples, I'll leave this as "Aeotec"